src/test/java/seedu/address/model/card/ScheduleTest.java

```diff
@@ -0,0 +1,31 @@
+package seedu.address.model.card;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+public class ScheduleTest {
+
+    @Test
+    public void isFeedbackSuccessful_getSuccessRate() {
+        Schedule s = new Schedule();
+        s.isFeedbackSuccessful(true);
+        s.isFeedbackSuccessful(false);
+        s.isFeedbackSuccessful(true);
+        assertEquals(s.getSuccessRate(), 2.0 / 3.0, 1);
+    }
+
+    @Test
+    public void isFeedbackSuccessful_getEasingFactor() {
+        Schedule s = new Schedule();
+        assertEquals(s.getEasingFactor(), 1.0, 3);
+        s.isFeedbackSuccessful(true);
+        assertEquals(s.getEasingFactor(), 1.0, 3);
+        s.isFeedbackSuccessful(false);
+        assertEquals(s.getEasingFactor(), 1.0, 3);
+        s.isFeedbackSuccessful(true);
+        assertEquals(s.getEasingFactor(), 1.0, 3);
+        s.isFeedbackSuccessful(true);
+        assertEquals(s.getEasingFactor(), 0.5649254682876186, 3);
```

Comment:
What is the significance of this double? Also is this saying that this can differ by up yo a plusminus 3?

Reply:
should be 0.001 I thought it was decimal places